packages/expo-cli/src/credentials/provider/iOSCredentialsProvider.ts

```diff
@@ -77,14 +82,29 @@ export default class iOSCredentialsProvider implements CredentialsProvider {
     return await credentialsJsonReader.readIosCredentialsAsync(this.projectDir);
   }
   private async getRemoteAsync(): Promise<iOSCredentials> {
-    await runCredentialsManager(this.ctx, new SetupIosBuildCredentials(this.app));
+    if (!this.app.skipCredentialsCheck) {
+      log('Skipping credentials check');
```

Comment:
Something seems wrong with this `if`. Either the message is inaccurate or the condition should be `if (this.app.skipCredentialsCheck))`.